<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Gill" ignored="0" name="Validating the title on the page" tests="2" failures="2" timestamp="2022-08-26T23:52:42 EDT" time="6.404" errors="0">
  <testcase name="basePageNavigation" time="0.032" classname="qaAcademy.HomePageTest">
    <failure type="java.lang.AssertionError" message="expected [FEATURED COURSE] but found [FEATURED COURSES]">
      <![CDATA[java.lang.AssertionError: expected [FEATURED COURSE] but found [FEATURED COURSES]
at qaAcademy.HomePageTest.basePageNavigation(HomePageTest.java:36)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- basePageNavigation -->
  <testcase name="basePageNavigation" time="0.035" classname="qaAcademy.HomePageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at qaAcademy.listeners.onTestFailure(listeners.java:35)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- basePageNavigation -->
</testsuite> <!-- Validating the title on the page -->
